notebooks/Customer_Automotive_Churn_Predictions.ipynb

```diff
@@ -0,0 +1,1866 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x7fac6bfbdb00>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from xgboost import plot_importance\n",
+    "import matplotlib\n",
+    "import matplotlib.pyplot as plt\n",
```

Comment:
`matplotlib` isn't listed in the conda env, so this import might fail.

Reply:
added it to env